civictechprojects/forms.py

```diff
@@ -56,3 +58,44 @@ def create_project(request):
             thumbnail_json = json.loads(project_thumbnail_location)
             thumbnail = ProjectFile.from_json(project, FileCategory.THUMBNAIL, thumbnail_json)
             thumbnail.save()
+
+    @staticmethod
+    def edit_project(request, project_id):
+        project = Project.objects.get(id=project_id)
+
+        if not request.user.username == project.project_creator.username:
```

Comment:
This seems like a potential security hole, because a bad actor can spoof the username in the request. Should you instead use some django built-ins to check if tokens are valid / if django claims the user is authorized to perform this action?

Reply:
My understanding is that request.user is populated from the session cookie.  And indeed, looking at the request headers I can't see anything that contains user information in plain text.